### ISSUE TITLE
Ecosystem path ID is not useful

### ISSUE THREAD

**Issue by wdduncan** [microbiomedata/pilot#14]
Displaying the Ecosystem path ID is not very useful. These path IDs are tied to unique 5 tuples of ecosystem category information. We to link to this 5 tuples. Perhaps we can navigate them using some like what is suggested in #13

**Comment by jeffbaumes** [comment]
Yes, I see that field on the chopping block for removal from faceting or item detail display. As part of having some sort of hierarchical selection of ecosystem this path ID could be removed.
